bot/start.ts

```
@@ -3,7 +3,8 @@ import { Telegraf, session, Context, Telegram } from 'telegraf';
 import { I18n, I18nContext } from '@grammyjs/i18n';
 import { Message } from 'typegram';
 import { UserDocument } from '../models/user';
-import { FilterQuery } from 'mongoose';
+import mongoose from 'mongoose';
+const { ObjectId } = mongoose.Types;
```

ESLint

[error] 'ObjectId' is assigned a value but never used. Allowed unused vars must match /^_/u.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'ObjectId' is assigned a value but never used.

(no-unused-vars)